generated/nidmm/nidmm/_converters.py

```diff
@@ -216,20 +216,18 @@ def convert_to_nitclk_session_number(item):
     Supported objects are:
     - class with .tclk object of type nitclk.SessionReference
     - nitclk.SessionReference
-    - NI-TClk Session Num
     '''
     try:
-        return item.tclk._get_session_number()
+        return item.tclk._get_tclk_session_reference()
```

Comment:
We are no longer adding Error classes that aren't used to the drivers.
We know which drivers support NI-TClk.
We could skip codegen of these converters trivially.

What do you think? I don't care much since unlike error classes, this is not user visible.